src/views/Rules/Rule/index.jsx

```diff
@@ -519,11 +519,14 @@ export default function Rule({ isNewRule, ...props }) {
             <Grid item xs={12} sm={6}>
               <TextField
                 fullWidth
+                select
                 label="Update Type"
-                value={defaultToEmptyString(rule.update_type)}
+                value={rule.update_type || 'minor'}
                 name="update_type"
-                onChange={handleInputChange}
-              />
+                onChange={handleInputChange}>
+                <MenuItem value="minor">minor</MenuItem>
+                <MenuItem value="major">major</MenuItem>
```

Comment:
We could capitalize dropdown values:

```jsx
<MenuItem value="minor">Minor</MenuItem>
<MenuItem value="major">Major</MenuItem>
```

Reply:
I think it's best to keep these lowercase actually, as they end up in the update XML (eg: https://aus5.mozilla.org/update/3/Firefox/33.0/20141202185629/Darwin_x86_64-gcc3-u-i386-x86_64/en-US/release/default/default/default/update.xml). (I realize we can have the label and value set differently, I just think it might be confusing/surprising to people).